charactersheet/charactersheet/templates/dm/encounter_sections/environment_section.tmpl.html

```diff
@@ -0,0 +1,73 @@
+<div class="container-fluid">
+  <div class="h3">
+    Environment<br />
+    <small>Where is this encounter, and what's it like there?</small>
+  </div>
+  <!-- Begin Tabs -->
+  <ul class="nav nav-tabs tabs">
+    <li role="presentation" data-bind="click: selectPreviewTab, css: previewTabStatus">
+      <a href="#" aria-controls="itemsModalPreview" role="tab" data-toggle="tab">
+          <b>Preview</b>
+      </a>
+    </li>
+    <li role="presentation" data-bind="click: selectEditTab, css: editTabStatus">
+      <a href="#" aria-controls="itemsModalEdit" role="tab" data-toggle="tab">
+          <b>Edit</b>
+      </a>
+    </li>
+  </ul>
+  <div class="tab-content">
+    <div role="tabpanel" data-bind="css: previewTabStatus" class="tab-pane">
+      <div class="row row-padded">
+        <div data-bind="col-xs-12">
+          <div class="row row-padded">
```

Comment:
I'd rather it either start on the edit or show some placeholder text. If we don't show it at all then that section is just a header.